package.json

```diff
@@ -23,16 +24,49 @@
   },
   "homepage": "https://github.com/buildo/state#readme",
   "dependencies": {
-    "rxjs": "^5.4.2"
+    "history": "^4.6.3",
+    "lodash": "^4.17.4",
+    "prop-types": "^15.5.10",
+    "qs": "^6.5.0",
+    "rxjs": "^5.4.2",
+    "tcomb": "^3.2.22"
+  },
+  "peerDependencies": {
+    "react": "^15"
   },
   "devDependencies": {
-    "babel-cli": "^6.22.2",
-    "babel-preset-buildo": "^0.1.1",
-    "jest": "^18.1.0",
-    "react": "^0.14.8",
-    "scriptoni": "^0.4.3",
+    "@types/debug": "0.0.30",
+    "@types/history": "^4.6.0",
+    "@types/jest": "^20.0.7",
+    "@types/lodash": "^4.14.73",
+    "@types/prop-types": "^15.5.1",
+    "@types/qs": "^6.5.0",
+    "@types/react": "^15",
+    "@types/react-dom": "^15",
+    "@types/react-test-renderer": "^15.5.2",
+    "jest": "^20.0.4",
+    "prettier": "^1.5.3",
+    "react": "^15",
+    "react-dom": "^15",
+    "react-test-renderer": "^15.6.1",
     "sinon": "^1.17.4",
     "smooth-release": "^6.0.0",
-    "tcomb": "^3.0.0"
+    "ts-jest": "^20.0.10",
+    "typelevel-ts": "^0.1.3",
+    "typescript": "^2.4.2"
+  },
+  "jest": {
+    "transform": {
+      "^.+\\.(t|j)sx?$": "<rootDir>/node_modules/ts-jest/preprocessor.js"
+    },
+    "testRegex": "(/__tests__/.*|\\.(test|spec))\\.(tsx?|jsx?)$",
+    "moduleFileExtensions": [
+      "ts",
+      "tsx",
+      "js",
+      "json",
+      "jsx"
+    ],
+    "testURL": "http://test.example.com"
```

Comment:
I was testing with browser history and wanted to have a valid path to be able to access the browser location from jsdom. I ended up manually mocking since I realized jest (and jsdom) don't support this at all. Will remove